packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -9,12 +9,21 @@ import { useStyles2, useTheme2 } from '../../themes';
  * @internal
  */
 export interface PanelChromeProps {
+  title?: string;
+  items?: {
+    orderedList: ReactNode[];
+    position?: 'left' | 'right'; // Default is 'left'
+  };
+  actionItems?: {
+    orderedList: ReactNode[];
+    position?: 'left' | 'right'; // Default is 'left'
+    show?: 'always' | 'hover' | 'never'; // Default is 'hover'
+  };
   width: number;
   height: number;
-  title?: string;
   padding?: PanelPadding;
-  leftItems?: React.ReactNode[]; // rightItems will be added later (actions links etc.)
-  children: (innerWidth: number, innerHeight: number) => React.ReactNode;
+  leftItems?: ReactNode[]; // rightItems will be added later (actions links etc.)
```

Comment:
If we're changing the API, please avoid naming properties like 'leftItems'. 
 
Evening ignoring right-to-level concerns, I think it's better API to give this more conceptual names that describes what should go into them (like primary vs secondary actions, or )

Reply:
yea, agree. We should try to figure out what the high level concepts are so we have some freedom to change/tweak design without the API becoming obsolete

Reply:
Yes, absolutely agree as well. We just have to first label "deprecated" the `leftItems` name for something else and change it for Grafana 10. At least I guess that's the way to go.

Reply:
Or, leave the one in grafana-ui as-is (you can deprecate it), and implement the new one _outside_ of grafana-ui in core grafana? Briefly looking, it doesn't even look like anyone uses it from grafana-ui?

Reply:
Hopefully, the new scene (new dashboard architecture) should make using PanelChrome directly not a common use case. But not sure.

Reply:
🤔 Regarding the implementation of these new layouts in the `Chrome Panel` of `grafana-ui`, it was [suggested](https://github.com/grafana/grafana/issues/56163#issuecomment-1266721257) by Torkel when we did our epic planning.

We assumed we wanted to give plugin app developers the ability to use this component to support rendering panels with the same look and feel as we have in Core Grafana, but just to check, was this the real reason, @torkelo?

Reply:
yes that is a general idea. But not 100% sure anymore. I think most future app plugins that want to visualize data will likely use the new scenes framework (still in the works/not exposed to plugins) and will not have to use PanelChrome directyl. But even then if they want to create their own panel-like thing (and not use an existing panel plugin) they will need to use something like PanelChrome from their custom Scene component. 

thoughts @dprokop ?

Reply:
I agree, I think if we strive to allow dashboard-like experiences in the apps, then PanelChrome should be exposed externally so that ScenesObjects can use it. Problem we need to figure out is the versioning again. If PanelChrome is exposed from grafana-ui, the Scenes would have to have their own version of grafana-ui bundled in order to provide stable experience - suboptimal IMO. So I think there's a chance Scenes may have to have it's own PanelChrome implemented, maybe a limited one, yet sharing the common look.